submissions/madmaxWMFU/Friends-App/js/app.js

```diff
@@ -0,0 +1,100 @@
+const allFriends = {
+    currentList: [],
+    changeList: [],
+    countFriends: 52,
+    minAge: 0,
+    maxAge: 150
+};
+
+const friendZone = document.querySelector(".friends-zone");
+const filters = document.querySelector(".filters-input");
+const sorts = document.querySelector(".search-sort-options");
+const resetFilters = document.querySelector(".reset-button");
+
+const getDataApi = async () => {
```

Comment:
Also problem with naming. From function name I conclude, that you will receive all API as some entity, not some concrete data.

Please, find better naming.